templates/github/.ci/scripts/schema.py.j2

```diff
@@ -0,0 +1,23 @@
+"""
+Customizing OpenAPI validation.
+
+OpenAPI requires paths to start with slashes:
+https://spec.openapis.org/oas/v3.0.3#patterned-fields
+
+But some pulp paths starts with curly brackets e.g. {artifact_href}
+This script modifies drf-spectacular schema validation to accept slashes and curly brackets.
+"""
```

Comment:
:+1:

Reply:
I do not see any templating in this file. Maybe ditch the ".j2" or include the dont-touch-this header.